server/pulp/server/db/model/__init__.py

```diff
@@ -667,6 +665,20 @@ def pre_save_signal(cls, sender, document, **kwargs):
         if not document._storage_path:
             document._storage_path = FileStorage.get_path(document)
 
+    def set_file(self, name):
```

Comment:
I feel like the names "set_file" and "import_content" are a little confusing together. They both sound like they might do the same thing (based only on the names). Do you think we could come up with more descriptive names so it's more obvious which one developers should use in which cases? I don't have great suggestions so this is a dumb complaint. Here's some terrible brainstorming:

set_complete_unit_file()
import_complete_content() # Maybe we could allow this to be used for any number of files but they must all be present?
set_file_on_content_unit_that_only_has_one_file_and_can_never_have_any_more()

OK I'm done. It's no big deal if you stay with set_file(), but if you can think of a way to better distinguish these names I think that would be helpful to developers.